build/patches/Enable-IntentBlockExternalFormRedirectsNoGesture.patch

```diff
@@ -2,6 +2,7 @@ From: csagan5 <32685696+csagan5@users.noreply.github.com>
 Date: Tue, 25 May 2021 19:33:41 +0200
 Subject: Enable IntentBlockExternalFormRedirectsNoGesture
 
+License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
```

Comment:
sure about this patch?
I modified it like this: https://github.com/uazo/bromite/blob/next-103/build/patches/Enable-IntentBlockExternalFormRedirectsNoGesture.patch

Reply:
The feature flag will not work?

Reply:
sorry, I didn't realize you did a revert.

I think `blockExternalFormRedirectsWithoutGesture` and `shouldDisableExternalIntentRequestsForUrl` do different things. I will check.

Reply:
Please let me know about this, I will merge this PR meanwhile.

Reply:
I double checked, my modification always blocks the opening regardless of the usergesture.
it is more correct as it is now.

Reply:
Ok, thanks - so I will not make any other change related to this.